build.sbt

```diff
@@ -14,84 +14,106 @@
  * limitations under the License.
  */
 
-name := "hyperspace-core"
+import Dependencies._
 
-sparkVersion := "2.4.2"
+lazy val spark24 = Version(2, 4, 2)
```

Comment:
Not in this change, but we could update Spark2 version to 2.4.4 or higher one. WDYT? @imback82